docs/users/cli-usage.md

```diff
@@ -277,7 +277,7 @@ ddev start
 ddev composer create --repository=https://repo.magento.com/ magento/project-community-edition
 ddev ssh
 bin/magento setup:install  --db-host=db --db-name=db --db-user=db --db-password=db  --admin-firstname=Magento --admin-lastname=User --admin-email=user@example.com  --admin-user=admin --admin-password=admin123 --language=en_US
-bin/magento setup:store-config:set --base-url="https://magento2.ddev.site/"
+bin/magento setup:store-config:set --base-url="https://magento2-ddev.ddev.site/"
```

Comment:
If you had created a directory named "magento2" instead of one named "magento2-ddev", it would have been right, I think. 

Probably the best fix here is to mention that. Add "mkdir magento2" to the front end, probably use `ddev config --project-type=magento2 --docroot=pub --create-docroot=true --project-name=magento2` to be even more explicit.

Reply:
@rfay you are right. I missed that default directory name was different on my side. I've added --project-name to be more explicit as you said. It will be easier for new users to test ddev without going deeply into documentation